package.json

```diff
@@ -45,6 +46,12 @@
       "<rootDir>/src/**/+(*.)+(spec|test).+(ts|js)?(x)"
     ]
   },
+  "lint-staged": {
+    "*.{ts,js,json,css,md}": [
+      "npm run format:fix",
```

Comment:
Won't this run prettier on the whole code base on every file change?
Imho this should just be `prettier --write`.
[Prettier config](https://prettier.io/docs/en/precommit.html#option-1-lint-staged-https-githubcom-okonet-lint-staged)

Reply:
Good catch, fixed! 😄